iterators.tex

```diff
@@ -6349,8 +6356,8 @@
 \oldtxt{\tcode{begin} and \tcode{end} are required to be amortized constant time.}
 
 \pnum
-\newtxt{Given an object \tcode{t} of type \tcode{T}, \oldtxt{the type \tcode{T} models}
-\tcode{Range\newtxt{<T>()}} \newtxt{is satisfied} if and only if}
+\newtxt{Given a value \tcode{t} of type \tcode{T}, \tcode{Range<T>()} is satisfied
+if and only if}
```

Comment:
Also, you have mismanaged the markup here. The change from the old "models" text to the new "satisfies" text has not been fully reviewed.